entries/t/truist.com.json

```diff
@@ -8,7 +8,7 @@
       "email"
     ],
     "custom-software": [
-      "Truist Authenticator"
+      "Custom app doesn't work - see reviews at https://play.google.com/store/apps/details?id=com.truist.authenticator and https://apps.apple.com/us/app/truist-authenticator/id1568427655"
```

Comment:
This should not be in the `custom-software` array. This should be in the [`notes` string](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#exceptions--restrictions).

However, I don't think this should be included at all. If the app is nonfunctional, it should be removed.

Reply:
That sounds good to me. I don't know about the note though.

@2factorauth/maintainers, any advice?

Reply:
Yeah sounds good :+1: